docs/dev/setup/jenkins-gitlab.rst

```diff
@@ -418,42 +405,12 @@ Start Jenkins
            continuous-integration:
                user: your.chosen.username
                password: your.chosen.password
-
-12. Setup JDK 15 in Jenkins Settings
-
-    Navigate in your browser into Jenkins → Manage Jenkins → Global Tool
-    Configuration → JDK. Change the existing JDK installation or click
-    on Add JDK.
-
-    Use ``OpenJDK 15`` as Name and
-    ``/usr/lib/jvm/java-15-openjdk-amd64`` as JAVA_HOME
-
-   .. figure:: jenkins-gitlab/jenkins_jdk_config.png
-      :align: center
-
 Required Jenkins Plugins
```

Comment:
add a new line to fix the GitHub action warning

Reply:
Fixed